src/style.css

```diff
@@ -1 +1,75 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Heavy.ttf');
+  font-weight: bold;
+}
+
+body {
+  padding: 0;
+}
+
+.search-bar {
+  margin: 20px 0;
```

Comment:
Be consistent with your margins. Add only top or bottom margin, but not both, to avoid potential margin collapse issues. In this case, you can consider using only `margin-bottom` or `margin-top` for the `.search-bar` class.